### DISCUSSION TITLE
Bug: Favicon reverts to Vercel's default on page refresh

### DISCUSSION THREAD

**Discussion by evolfo** [vercel/next.js#22555] [Help]
I'm running into an issue that I haven't seen mentioned anywhere else. I have a next.js site up and running, and the custom favicon shows up fine. However, if I refresh the page, the favicon switches to the Vercel default, and will stay until I navigate somewhere else. 

I'm importing the favicon as such in between <Head> tags in my index.js file: 
`<link rel="shortcut icon" href="/img/favicon.ico" />`

Any ideas as to what could be causing this and some potential solutions? 

Thank you!

**Comment by jamesmosier** [comment]
Hi @evolfo. This sounds like the favicon is just cached in your browser. If you open up a private browsing tab or a completely different browser does the issue persist? 

Also an alternative place to put your favicon code would be in a [custom Document](https://nextjs.org/docs/advanced-features/custom-document) (`_document.js`). This would ensure the favicon is rendered on every page.

**Reply by evolfo** [reply]
I don't think that it being cached is the issue, as it persists in a private browser window. It renders correctly until a manual refresh happens, then it shows the vercel logo. Thank you, I'll try the custom document and see what happens.
